apps/web/.env.local.example

```
@@ -28,4 +28,7 @@ AUTH_SERVER_BASE_URL=
 
 # LiveState
 LIVE_STATE_WS_URL=
-LIVE_STATE_API_URL=
\ No newline at end of file
+LIVE_STATE_API_URL=
+
+# Reflag
+VITE_REFLAG_PUB_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)